wsd/ClientSession.cpp

```diff
@@ -517,7 +517,7 @@ bool ClientSession::_handleInput(const char *buffer, int length)
                 endptr = nullptr;
                 double counter = strtod(str, &endptr);
                 if (*endptr == '\0' && counter > 0 &&
-                    (counter < (double)(std::numeric_limits<uint64_t>::max() / 1000)))
+                    (counter < (uint64_t)(std::numeric_limits<uint64_t>::max() / 1000)))
```

Comment:
Hrm, turns out this now breaks the clang build:

```
wsd/ClientSession.cpp:520:32: error: implicit conversion from 'uint64_t' (aka 'unsigned long') to 'double' changes value from 18446744073709551 to 18446744073709552 [-Werror,-Wimplicit-const-int-float-conversion]
                    (counter < (uint64_t)(std::numeric_limits<uint64_t>::max() / 1000)))
                             ~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [Makefile:2509: wsd/ClientSession.o] Error 1
make[2]: *** Waiting for unfinished jobs....
````

Would `(double)(uint64_t)` work for Coverity? That works to keep clang happy and I think it's not an actual change in behavior (just making that conversion to double explicit).